hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -1 +0,0 @@
-module com.hedera.services.api.implementation {}
```

Comment:
Why have you removed the module info?

Reply:
`hedera-app` needs to depend on `hedera-mono-service`, which is not a module. Fixing this is out of scope for this PR. Since there is no valuable information in the definition at this point, I think the easiest is to remove it for now and add it later once everything is fixed.